### ISSUE TITLE
raise JSONDecodeError("Expecting value", s, err.value) from None

### ISSUE THREAD

**Issue by Tim55667757** [Tim55667757/TKSBrokerAPI#18]
Add a clear notification when `raise JSONDecodeError("Expecting value", s, err.value) from None` trace when there is no connection.

```text
Traceback (most recent call last):
  File "TKSBrokerAPI.py", line 2065, in <module>
    Main()
  File "TKSBrokerAPI.py", line 1895, in Main
    server = TinkoffBrokerServer(args.token, instrumentsList=instruments)  # Init class for trading with Tinkoff Broker
  File "TKSBrokerAPI.py", line 104, in __init__
    self.instrumentsList = instrumentsList if instrumentsList is not None else self.Listing()  # dictionary with stocks, currencies, bonds and etfs supports by broker. You can define this param whwn class init to reduce server requests.
  File "TKSBrokerAPI.py", line 275, in Listing
    instrumentsList = self._SendRequest(stocksURL, debug=showDebug)["payload"]["instruments"]
  File "TKSBrokerAPI.py", line 191, in _SendRequest
    responseJSON = self._ParseJSON(response.text, debug=False)
  File "TKSBrokerAPI.py", line 125, in _ParseJSON
    responseJSON = json.loads(rawData, encoding="UTF-8")
  File "c:\python3\lib\json\__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "c:\python3\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "c:\python3\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by Tim55667757** [comment]
Fixed in latest 1.3.* builds.

Error handling added for `raise JSONDecodeError("Expecting value", s, err.value) from None`. Now the message is showing: `Check you Internet connection! Failed to establish a new connection to broker server!` and the path to the debug log file.

**Comment by Tim55667757** [comment]
Debug build for this issue: tksbrokerapi 1.3.dev67.
PyPI: https://pypi.org/project/tksbrokerapi/1.3.dev67/
